packages/manager/src/managers/prismicRepository/PrismicRepositoryManager.ts

```diff
@@ -174,10 +174,9 @@ export class PrismicRepositoryManager extends BaseManager {
 		const text = await res.text();
 
 		// Endpoint returns repository name on success, which must be more than 4 characters and less than 30
-		// if (!res.ok) {
-		if (!res.ok || text.length < 4 || text.length > 30) {
```

Comment:
@lihbr on this PR I removed a strange condition you added.

When the back-end return that the creation was ok, the name should be well formatted and even if it's not a good one, it's a success anyway. Strange to fail when back-end return that it's ok, no?

If you can review this PR at the same time, I would like to have the detailled error message with your release

Reply:
I added that back then because the backend returns 200 OK when it actually encounters 500 (IIRC it happens when the auth state is expired/invalid)

The only "real" 200 OK for this call is when these conditions are all met:
- Response status is 200
- Response text is the created repository ID (which can sometime differ from the one send since the backend also slugifies the ID to some extends on its end)

All other responses are errors. So I guess the condition could be updated to check that `text.length <= 63` (subdomain length limit)?

Reply:
You actually can easily test that by sending a POST like this:
```
POST /authentication/newrepository?app=slicemachine HTTP/2
Host: prismic.io
user-agent: insomnia/2023.4.0
accept: */*
content-length: 108

{
	"domain": "test",
	"framework": "nuxt",
	"plan": "personal",
	"isAnnual": "false",
	"role": "developer"
}
```

Reply:
Thank you for the explanation!